From: Bryan Donlan <bdonlan@gmail.com>
Date: Thu, 12 May 2011 00:10:57 -0400
Subject: [PATCH] Reference the m4 directory for macro files

This patch adds references to the m4/ directory in configure.in
(AC_CONFIG_MACRO_DIR) and Makefile.am (ACLOCAL_AMFLAGS); these are
required for libtoolize to work properly
---
 Makefile.am  |    2 ++
 configure.in |    2 +-
 2 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index ed3ff6f..3a83560 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,3 +1,5 @@
+ACLOCAL_AMFLAGS = -I m4
+
 if BUILD_OFXCONNECT
   MAYBE_OFXCONNECT = ofxconnect
 endif
diff --git a/configure.in b/configure.in
index 22fe59f..ead721c 100644
--- a/configure.in
+++ b/configure.in
@@ -11,9 +11,9 @@ dnl Process this file with autoconf to produce a configure script.
 AC_INIT(inc/libofx.h.in)
 AM_CONFIG_HEADER(config.h)
 AC_CONFIG_AUX_DIR(config)
+AC_CONFIG_MACRO_DIR(m4)
 AC_PROG_CC
 AC_PROG_CXX
-#AC_PROG_RANLIB
 
 m4_include([libcurl.m4])
 
-- 
